test/unit/liquid/drops/alert_drop_test.rb

```diff
@@ -25,10 +25,6 @@ def setup
     assert_equal @alert.timestamp, @drop.timestamp
   end
 
-  test 'returns description' do
-    assert_equal @alert.message, @drop.message
```

Comment:
maybe make sure it is `my message`?

Reply:
btw, this was deleted, because there was another test with the same assertions:
```
 test 'returns friendly message' do
   assert_equal @alert.message, @drop.message
 end
```

Do you prefer to assert the explicit value?

Reply:
Thanks, leave it.